### PR TITLE
fix docs for how to change colors in REPL

### LINKED ISSUE #18471: Suggestion for editing prompt color in manual doesn't work.
http://docs.julialang.org/en/release-0.5/manual/interacting-with-julia/#customizing-colors

> To change the color of the Julia prompt you can add something like the following to your `juliarc.jl` file:
>
> ```jl
 Base.active_repl.prompt_color = Base.text_colors[:cyan]
```

This gives the error: `ERROR: LoadError: UndefVarError: active_repl not defined` because the REPL is not loaded until later.

### PR DESCRIPTION
**Pull request by KristofferC**
Fixes #18471 

cc @Keno

### CODE REVIEW

**Comment by KristofferC** [comment]
Can someone confirm that this works on their setup?

File: doc/manual/interacting-with-julia.rst [hunk 1]
```diff
@@ -289,13 +289,17 @@ Customizing Colors
...
+    end
+
+    atreplinit(customize_colors)
 
 The available color keys in ``Base.text_colors`` are ``:black``, ``:red``, ``:green``, ``:yellow``,
 ``:blue``, ``:magenta``, ``:cyan``, ``:white``, ``:normal``, and ``:bold``. Similarly, you can
 change the colors for the help and shell prompts and input and answer text by setting the
-appropriate member of ``Base.active_repl`` (respectively, ``help_color``, ``shell_color``,
-``input_color``, and ``answer_color``). For the latter two, be sure that the ``envcolors`` member
+appropriate field of ``repl`` in the `customize_colors` function above (respectively, ``help_color``, ``shell_color``,
```
Comment (tkelman) [inline doc/manual/interacting-with-julia.rst]: double backticks for rst

Comment (KristofferC) [inline doc/manual/interacting-with-julia.rst]: ok

**Comment by KristofferC** [comment]
This should be fairly easy to review and would be good to not have known faults in the docs for 0.5.0.

**Comment by tkelman** [comment]
Has anyone had time to verify that this is accurate and works in juliarc?

**Comment by KristofferC** [comment]
Well, I obviously tried it but always good with confirmations :P

**Pull request merged**